xdocs/usermanual/component_reference.xml

```diff
@@ -6521,17 +6521,39 @@ See <a href="get-started.html#classpath">JMeter's Classpath</a> and
        The users are generated according to the schedule.
     </p>
     <p>The load profile consists of a sequence of constant, increasing or decreasing load.
-       The following commands are available:
+       The basic configuration is <code>rate(1/sec) random_arrivals(2 min) rate(3/sec)</code> which means the load will increase linearly
+        from 1 request per second to 3 requests per second during 2-minute timeframe.
```

Comment:
```suggestion
        from one request per second to three requests per second during a period of two-minutes.
```
Numbers up to twelve should be spelled out.
I am not really sure about the paraphrasing at the end, but Firefox does not know the word *timeframe*.